### PR TITLE
Update v2.1.3 to v2.2.3 with the latest spec v2 changes.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by notlee** [comment]
This shouldn't have valid sass, so the failure is good and expected :)

**Review by JakeChampion — APPROVED** [review]

**Comment by notlee** [comment]
I force pushed here to update the readme, which had a linting warning due to the npm namespace in the heading title

**Review by JakeChampion — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ To learn about getting started with other Origami components see the [Origami co
 ## Versions
 
 |version|valid js|valid sass|valid html|valid demos|valid readme|js tests pass|sass tests pass|
-|-------|--------|----------|----------|-----------|------------|-------------|---------------|
+|***---|******|******--|******--|******---|*********|*********-|*********---|
```
Comment (JakeChampion) [inline readme.md:21]: This looks like a find and replace mistake as the table no longer has column headings

**Review by JakeChampion — APPROVED** [review]

**Review by JakeChampion — APPROVED** [review]

**Review by JakeChampion — APPROVED** [review]

**Comment by notlee** [comment]
tagged and released

**Pull request closed without merging**